Decorator/CarDecorator/BaseCar.cs

```diff
@@ -0,0 +1,18 @@
+﻿using System;
```

Comment:
Delete all others files form this PR, fork from master all branches

Reply:
done